contracts/deploy/035_reduce_collect_gas.js

```diff
@@ -0,0 +1,38 @@
+const { deploymentWithProposal } = require("../utils/deploy");
+
+module.exports = deploymentWithProposal(
+  { deployName: "035_reduce_collect_gas", forceDeploy: false },
+  async ({
+    assetAddresses,
+    deployWithConfirmation,
+    ethers,
+    getTxOpts,
+    withConfirmation,
+  }) => {
+    const { deployerAddr, governorAddr } = await getNamedAccounts();
```

Comment:
nit: I think line 12, 13 can be removed - those vars are not used?

Reply:
Thanks, fixed these.